src/esssans/normalization.py

```diff
@@ -102,14 +103,20 @@ def transmission_fraction(
     return TransmissionFraction(frac)
 
 
-def iofq_denominator(
+_broadcasters = {
+    UncertaintyBroadcastMode.drop: drop_variances_if_broadcast,
+    UncertaintyBroadcastMode.upper_bound: broadcast_with_upper_bound_variances,
+    UncertaintyBroadcastMode.fail: lambda x, sizes: x,
+}
+
+
+def iofq_norm_wavelength_term(
     data_transmission_monitor: CleanMonitor[RunType, Transmission],
     direct_incident_monitor: CleanMonitor[DirectRun, Incident],
     direct_transmission_monitor: CleanMonitor[DirectRun, Transmission],
-    solid_angle: SolidAngle[RunType],
     direct_beam: Optional[CleanDirectBeam],
     uncertainties: UncertaintyBroadcastMode,
-) -> Clean[RunType, Denominator]:
+) -> NormWavelengthTerm[RunType]:
     """
     Compute the denominator term for the I(Q) normalization. This is basically:
```

Comment:
So now you have `iofq_norm_wavelength_term` and `iofq_denominator`. Is this docstring more for the latter function?

Reply:
Thanks, see update!